src/dev_server.rs

```diff
@@ -58,7 +58,7 @@ use std::{
 #[clap(
     about = "A simple HTTP server useful during development.",
     long_about = "A simple HTTP server useful during development.\n\
-        It can watch the source code for changes.",
+        It can watch the source code for changes."
```

Comment:
If we want to add something to `#[clap()]`, the diff will add a useless line

```suggestion
        It can watch the source code for changes.",
```

Reply:
rustfmt removed it. not me. I dont care

Reply:
you really thought i would have go out of my way to fix this xD

Reply:
I'm trying to be a worst person